maintenance/kubernetes/aks-1.22.md

```diff
@@ -151,7 +151,16 @@ Related to `prodpublicks8` and `temp-privatek8s` clusters
 
         - [x] Expected downtimes: none
 
-      ### Notes
+      ### BUG NOTE AFTER UPGRADE
+        
+        ERROR : we have to specify the namespace where the SAS token lives, otherwises Kubernetes searches on the default namespace.
```

Comment:
```suggestion


#### Persistent Volumes of kind "Azurefile"

ERROR : we had to specify the namespace where the SAS token lives, otherwises Kubernetes searches on the default namespace.
```

reason: SAS token is only for Azurefile (the Azuredisk doesn ot have such concern